kong/kong.lua

```diff
@@ -239,6 +240,13 @@ function Kong.balancer()
       tostring(addr.ip),"' port:",tostring(addr.port),"): ", tostring(err))
     return responses.send_HTTP_INTERNAL_SERVER_ERROR()
   end
+
+  ok, err = set_timeouts(addr.connect_timeout / 1000,
+                         addr.send_timeout / 1000,
+                         addr.read_timeout / 1000)
```

Comment:
Can't we do these calculations at config-load time, instead of on each request? Should be an easy fix with the new caching (the way I intend to do it)